consensus/src/round_manager.rs

```diff
@@ -245,9 +248,10 @@ impl RoundManager {
             .proposer_election
             .is_valid_proposer(self.proposal_generator.author(), new_round_event.round)
         {
-            let proposal_msg = self.generate_proposal(new_round_event).await?;
+            let proposal_msg =
```

Comment:
Should we even broadcast a proposal if consensus should not make progress?  It seems like it would be faster to halt progress by avoiding proposing as well.

Reply:
doesn't matter much imo, we only broadcast once in a round

Reply:
I'm not concerned about the extra traffic, I'm just saying that if the goal is to stop consensus from progressing as fast as possible, getting rid of proposals in addition to votes will slow down the commit rate - rounds when the sync_only validator is leader will never get a QC.